apis/kueue/v1beta1/workload_conversion_test.go

```diff
@@ -0,0 +1,229 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/ptr"
+
+	"sigs.k8s.io/kueue/apis/kueue/v1beta2"
+)
+
+func TestConvert_v1beta1_WorkloadStatus_To_v1beta2_WorkloadStatus(t *testing.T) {
+	testCases := map[string]struct {
+		input    *WorkloadStatus
+		expected *v1beta2.WorkloadStatus
```

Comment:
```suggestion
		input    *Workload
		expected *v1beta2.Workload
```
I’m wondering if we can combine spec and status into a single test. We could probably use Convert_v1beta1_Workload_To_v1beta2_Workload for that.

Reply:
Or event more use ConvertTo(). In that case you don't need to create separate TestWorkloadConversion_RoundTrip test.